Android/app/src/main/java/com/safehello/safehello_android_sdk_samples/MainActivity.kt

```diff
@@ -23,67 +25,78 @@ class MainActivity : AppCompatActivity() {
         private const val TAG = "MainActivity"
         private const val HOST_USER_ID = "host-id"
         private const val PARTICIPANT_USER_ID = "participant-id"
+        private var HOST_TOKEN = ""
+        private var PARTICIPANT_TOKEN = ""
     }
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
-
         val progressLayout = findViewById<View>(R.id.progressLayout)
         progressLayout.visibility = View.VISIBLE
-        connectToSafeHello()
+
+        getUserToken(HOST_USER_ID)
```

Comment:
@guilhermekrz may be able to cleanup these two calls to `getUserToken`; they are almost identical, and it would be nice to remove duplicated code.